site/docs/uk/guide/deployment-types.md

```diff
@@ -226,7 +226,27 @@ app.use(webhookCallback(bot, "oak"));
 
 > Зауважте, що ви не повинні викликати `bot.start()` при використанні вебхуків.
 
-Обовʼязково прочитайте [чудовий путівник Марвіна про все, що стосується вебхуків](https://core.telegram.org/bots/webhooks), написаний командою Telegram, якщо ви плануєте запустити свого бота на вебхуках у VPS.
+Тепер ваш додаток слухає запити на вебхук від Telegram.
+Останнє, що вам потрібно зробити, це вказати Telegram, куди надсилати оновлення.
+Існує кілька способів зробити це, але в кінцевому підсумку всі вони просто викликають `setWebhook`, як описано в документації [тут](https://core.telegram.org/bots/api#setwebhook).
+
+Найпростіший спосіб встановити вебхук --- це вставити наступну URL-адресу в адресний рядок браузера, замінивши `<токен-бота>` на токен вашого бота, а `<URL-адреса>` --- на публічну кінцеву точку вашого сервера.
+
+```txt
+https://api.telegram.org/bot<токен-бота>/setWebhook?url=<URL-адреса>
+```
+
+Крім того, ми створили відповідний інтерфейс для цього, якщо ви віддаєте перевагу управлінню вашим вебхуком через вебсайт.
+Ви можете знайти його тут: <https://telegram.tools/webhook-manager>
+
+Зверніть увагу, що ви також можете налаштувати свій вебхук прямо в коді:
+
+```ts
+const endpoint = ""; // <-- вкажіть вашу URL-адресу тут
+await bot.api.setWebhook(endpoint);
+```
+
+Нарешті, обовʼязково прочитайте [чудовий путівник Марвіна про все, що стосується вебхуків](https://core.telegram.org/bots/webhooks), написаний командою Telegram, якщо ви плануєте запускати свого бота на вебхуках на VPS.
```

Comment:
```suggestion
Нарешті, обовʼязково прочитайте [чудовий путівник Марвіна про все, що стосується вебхуків](https://core.telegram.org/bots/webhooks), написаний командою Telegram, якщо ви плануєте запускати свого бота на вебхуках.
```

Why VPS? What if Vercel, VDS, etc?

Reply:
The guide only applies to webhooks on a VPS. Vercel takes care of so many things for you that the guide isn't really needed. Also, if you find a mistake in the docs, please report it so we can fix the English version. Don't just put Ukrainian out of sync, even if it improves it :)

Reply:
> The guide only applies to webhooks on a VPS. Vercel takes care of so many things for you that the guide isn't really needed. Also, if you find a mistake in the docs, please report it so we can fix the English version. Don't just put Ukrainian out of sync, even if it improves it :)

I just propose how it can be improved and then @niusia-ua can checkI suggest a possible improvement in the translation and if it turns out to be really better, we will suggest it in the original text as well.

Reply:
Okay awesome!